cfme/common/provider_views.py

```diff
@@ -170,6 +170,27 @@ def is_displayed(self):
                 self.navigation.currently_selected == ['Middleware', 'Providers'])
 
 
+class ContainerProviderDetailsView(ProviderDetailsView):
```

Comment:
Is this coming from a merge? Don't think I should see it in this PR

Reply:
This was added as part of the rebase with master. I will rebase again today and pay attention to see if this should drop out.